test/deals.js

```diff
@@ -140,24 +260,82 @@ describe('Deals', function() {
           ],
         })
         .then(data => {
+          dealId = data.dealId
           expect(data).to.be.a('object')
         })
     })
   })
 
-  // describe('Associate', function () {
-  //   it('Returns a 204 response if successful.', function () {
-  //     return hubspot.deals.associate(1126609, 'CONTACT', 394455).then(data => {
-  //       expect(data).to.be.a('object')
-  //     })
-  //   })
-  // })
-
-  // describe('Remove Association', function () {
-  //   it('Returns a 200 response if successful.', function () {
-  //     return hubspot.deals.removeAssociation(1126609, 'CONTACT', 394455).then(data => {
-  //       expect(res.statusCode).to.equal(204)
-  //     })
-  //   })
-  // })
+  describe('Associate', function() {
+    let dealId = 123
+    let companyId = 234
+    const associationType = 'COMPANY'
```

Comment:
Just curious, what are the other possible association types?

Reply:
I think `CONTACT` is the only other type

Reply:
https://developers.hubspot.com/docs/methods/deals/associate_deal